TranslucentTB/main.cpp

```diff
@@ -16,32 +17,170 @@ struct WINCOMPATTRDATA
 	ULONG ulDataSize;
 };
 
+struct OPTIONS
+{
+	int mode;
```

Comment:
Could you change this from `mode` to something a bit more descriptive such as `blur` or something? Thanks!